configure.ac

```diff
@@ -74,6 +74,12 @@ fi
 AM_CONDITIONAL(ENABLE_TESTS_GLES,
     [test "x$enable_tests_gles" = "xyes"])
 
+HAVE_GTEST="no"
+ifdef([GTEST_LIB_CHECK],
```

Comment:
gtest doesn't provide a gtest.pc file.

Interestingly, I just found this:

https://code.google.com/p/googletest/wiki/FAQ#Why_is_it_not_recommended_to_install_a_pre-compiled_copy_of_Goog

...which implies that we should compile gtest during libyami compilation.  I'm not sure how we should manage that, however.  Would it be better to handle gtest as a git submodule, a tarball, a download-on-first-compile, or some other way?